ENIGMAsystem/SHELL/Widget_Systems/TinyFileDialogs/Info/About.ey

```diff
@@ -0,0 +1,9 @@
+%e-yaml
+---
+
+Name: Tiny File Dialogs
+Identifier: TinyFileDialogs
+Build-Platforms: MacOSX, Linux
```

Comment:
Does this not work on Windows?

Reply:
TFD does work on Windows, but I wanted to write my own implementation for windows which is more accurate to how GM works and will replace the Win32 dialogs system. I already wrote it, but as I've stated many times now there is a bug with MinGW that prevents my code from being compiled right now. It works with Microsoft's compiler, but not with MinGW. MinGW is missing a function I need for get_directory_alt(capt, root).

Reply:
I don't know whether you actually want me to magically change this problem somehow before you merge or you are just complaining it doesn't work on Windows yet.

Reply:
@JoshDreamland I don't know whether you actually want me to magically change this problem somehow before you merge or you are just complaining it doesn't work on Windows yet.

Reply:
I was just asking why you omitted Windows from the platforms list.